WebApiCoreSeed.WebApi/Controllers/ValuesController.cs

```diff
@@ -1,45 +1,52 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc;
-using Microsoft.AspNetCore.Authorization;
-
-namespace WebApiCoreSeed.WebApi.Controllers
+﻿namespace WebApiCoreSeed.WebApi.Controllers

```

Comment:
I don't know if there's a standard for this project but we usually leave the usings before the namespace (even though StyleCop may have a rule for that) and we leave only the ones being used. What do you think about that? cc @mravinale @mvazquez-ms @matiasbeckerle